websocketjs.html

```diff
@@ -14,7 +14,10 @@
 //onmessage writes everything incoming from ws to console
 socket.onmessage = function (evt) {
      var obj = JSON.parse(evt.data)
+    delete obj.body;
```

Comment:
why do we deliting body here?
and why do you think that our obj format is sutable for fetch? magic :)